wpimath/src/main/native/include/frc/kinematics/ChassisSpeeds.h

```diff
@@ -249,6 +265,17 @@ struct WPILIB_DLLEXPORT ChassisSpeeds {
   constexpr ChassisSpeeds operator/(double scalar) const {
     return operator*(1.0 / scalar);
   }
+
+  /**
+   * Compares the ChassisSpeeds with another ChassisSpeed.
+   *
+   * @param other The other ChassisSpeeds.
+   *
+   * @return The result of the comparison. Is true if they are the same.
+   */
+  constexpr bool operator==(ChassisSpeeds& other) {
+    return vx == other.vx && vy == other.vy && omega == other.omega;
+  }
```

Comment:
```suggestion
  constexpr bool operator==(ChassisSpeeds& other) = default;
```

Reply:
This should also take a `const ChassisSpeeds&`, not ` ChassisSpeeds&`.

Also, just to double check, we're good with strict double equality in this case? (I think it's fine since most cases will really just be checking if it's the exact same object, but I just want to double-check)

Reply:
Yea.